### DISCUSSION TITLE
Regarding failed github copilot payment

### DISCUSSION THREAD

**Discussion by suyashpatil78** [community/community#132043] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have made a purchase of github copilot using my card (I have been using this card for almost an year now) and the payment got failed as per the billing page (Error message was - The credit card was declined by the payment processor). However, money got deducted from my account. I have already raised a support ticket for this but there is not even a single update. Should I raise payment related issue elsewhere?

**Comment by RachidZahrani** [comment]
ontact Your Bank: Ensure no issues on their end.
Follow Up with GitHub Support: Send a polite reminder about your ticket.
GitHub Payment Issue Form: Check if GitHub has a specific form for payment issues on their support page.
Social Media: Sometimes contacting GitHub via Twitter or other social media can get a quicker response.
